### PR TITLE
feat: new TS db schema (VF-1305)

### PR DESCRIPTION
**Pull request by xavidop**
**Fixes or implements VF-XXX**

### Related PRs

| branch              | PR          |
| ------------------- | ----------- |
| other_pr_production | [link](url) |
| other_pr_master     | [link](url) |

### Checklist

- [ ] title of PR reflects the branch name
- [ ] all commits adhere to conventional commits
- [ ] appropriate tests have been written
- [ ] all the dependencies are upgraded

### CODE REVIEW

**Review by jonahsnider — APPROVED** [review]

File: lib/clients/analytics.ts [hunk 1]
```diff
@@ -1,13 +1,12 @@
 import Rudderstack, { IdentifyRequest, TrackRequest } from '@rudderstack/rudder-sdk-node';
```
Comment (frankgu968) [inline lib/clients/analytics.ts:1]: We should remove the rudderstack dependency altogether

File: lib/services/alexa/request/lifecycle/response.ts [hunk 2]
```diff
@@ -60,6 +61,8 @@ export const responseGenerator = (utils: typeof utilsObj) => async (runtime: Ale
     payload: response,
     sessionid: input.requestEnvelope.session?.sessionId,
     metadata: runtime.getFinalState(),
+    timestamp: new Date(),
```
Comment (frankgu968) [inline lib/services/alexa/request/lifecycle/response.ts:64]: Shouldn't this be serialized with a toString()?

File: lib/services/alexa/request/lifecycle/update.ts [hunk 3]
```diff
@@ -23,14 +23,16 @@ const update = async (runtime: AlexaRuntime, input: AlexaHandlerInput): Promise<
...
-  runtime.services.analyticsClient.track({
+  const turnID = await runtime.services.analyticsClient.track({
     id: runtime.getVersionID(),
-    event: Event.INTERACT,
+    event: Event.TURN,
     request: input?.requestEnvelope?.request?.type === 'LaunchRequest' ? RequestType.LAUNCH : RequestType.REQUEST,
     payload: runtime.getRequest(),
     sessionid: input.requestEnvelope.session?.sessionId,
     metadata: runtime.getFinalState(),
+    timestamp: new Date(),
```
Comment (frankgu968) [inline lib/services/alexa/request/lifecycle/update.ts:33]: toString() ?

File: lib/services/alexa/request/lifecycle/response.ts (hunk 2 above)
Comment (jonahsnider) [reply lib/services/alexa/request/lifecycle/response.ts:64]: gets serialized once when building request body

https://github.com/voiceflow/alexa-runtime/blob/795cea0df66c5b5ea54d3f97d66710fff520c3d0/lib/clients/analytics.ts#L100

File: lib/services/alexa/request/lifecycle/update.ts (hunk 3 above)
Comment (jonahsnider) [reply lib/services/alexa/request/lifecycle/update.ts:33]: see other comment

File: lib/services/alexa/request/lifecycle/response.ts (hunk 2 above)
Comment (xavidop) [reply lib/services/alexa/request/lifecycle/response.ts:64]: yup!

**Pull request merged**